core/modules/date/views/date_views_argument_handler_simple.inc

```diff
@@ -7,49 +7,10 @@
 /**
  * Date API argument handler.
  */
+// @codingStandardsIgnoreStart
 class date_views_argument_handler_simple extends views_handler_argument_date {
 
   /**
-   * @var date_sql_handler
-   */
-  var $date_handler;
-
-  /**
-   * @var string
-   */
-  var $format;
-
-  /**
-   * @var string
-   */
-  var $sql_format;
-
-  /**
-   * @var array
-   */
-  var $placeholders = array();
-
-  /**
-   * @var string
-   */
-  var $base_table;
-
-  /**
-   * @var string
-   */
-  var $original_table;
-
-  /**
-   * @var DateTime
-   */
-  protected $min_date;
-
-  /**
-   * @var DateTime
-   */
-  protected $max_date;
-
```

Comment:
Seems like all these variables should stay defined, unless they actually do not exist.